.github/labels.yml

```diff
@@ -1,38 +1,91 @@
-- name: bug
-  description: Something isn't working
+# The labels declared in this file are the ones avialables
+# https://github.com/ansys/ansys-sphinx-theme/labels
+
+# -- Defects and glitches labels----------------------------------------------
+
+- name: 'bug'
+  description: Defects or glitches reported by users or developers
   color: d42a34
 
-- name: dependencies
-  description: Related with project dependencies
-  color: ffc0cb
+- name: 'fix'
+  description:  Pull requests related to resolving problems or errors
+  color: d42a34
+
+# -- Style labels ------------------------------------------------------------
+
+- name: 'style'
+  description: Improvements related to general project style
+  color: dcbeff
+
+- name: 'style:code'
+  description: Improvements related to code style
+  color: dcbeff
+
+- name: 'style:docs'
+  description: Improvements related to doc style
+  color: dcbeff
+
+- name: 'style:branch-name'
```

Comment:
Can we add a branch name rule instead of having such label style ?

Reply:
Done.